app/views/catalog/_about_this_collection_block.html.erb

```diff
@@ -11,7 +11,10 @@
             <% if field_name == "finding_aid_link_ssm" %>
               <dd class="blacklight-<%= field_name.parameterize %> col-md-7"><a href="<%= doc_presenter.field_value field %>">Finding Aid</a></dd>
             <% else %>
-              <dd class="blacklight-<%= field_name.parameterize %> col-md-7"><%= doc_presenter.field_value field %></dd>
+              <dd class="blacklight-<%= field_name.parameterize %> col-md-7">
+                <%= dates_on_separate_lines(document, field_name).html_safe if field_is_for_dates?(field_name) %>
```

Comment:
Have you tried using an if-else statement here (and in the other block file)? It might be more readable and would prevent us from having to call the `field_is_for_dates?` method twice.

Reply:
I set it up with the if/else statement, and then refactored to reduce the lines used. I will bring it back for readability.

Reply:
Changes per request.